api/src/main/java/ca/bc/gov/educ/assessment/api/model/v1/AssessmentStudentEntity.java

```diff
@@ -32,6 +34,12 @@ public class AssessmentStudentEntity {
   @JoinColumn(name = "ASSESSMENT_ID", referencedColumnName = "ASSESSMENT_ID")
   AssessmentEntity assessmentEntity;
 
+  @ToString.Exclude
```

Comment:
I think we only want the ASSESSMENT_FORM_ID UUID here for this one so we don't fetch when pulling the students.